Automattic/pocket-casts-android
# Prioritize embedded chapters

## Description

This PR changes embedded chapters to be prioritized over remote ones. This is needed because ads injection can make remote chapter timestamps invalid.

Closes #2093 

## Testing Instructions

1. Fresh install the app from 06b94680454297485ed455f091bc4d5291fedd9a.
2. Play [this episode](https://pca.st/vg3xvczk) until you see chapters.
3. Update the app with changes from `task/prioritize-embedded-chapters`.
4. Open the app.
5. Run `SELECT * FROM episode_chapters WHERE episode_uuid IS 'dc063c76-b585-43c2-80b2-6413fa334a30'` in Database Inspector. You should see chapters with `is_embedded` equal to `0`.
6. Play the episode from the 2nd step.
7. After a while query from 6th step should show that `is_embedded` is equal to `1`.
8. Make sure that the you can see chapters and some of the chapters have custom artwork.

## Checklist
 
#### I have tested any UI changes...
